### PR TITLE
Symbol Lookup

### PR DESCRIPTION
**Pull request by aarushipandey**
Final Symbol Lookup Page 

SymbolLookup

### CODE REVIEW

**Comment by aarushipandey** [comment]
[](url)

**Review by christinem61 — DISMISSED** [review]

**Comment by christinem61** [comment]
It looks good! I think we just need to connect it to the individual stock page now. For next demo, maybe we should address what happens when incorrect ticker is entered?

File: frontend/src/components/Trade/Trade.js [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+                </div>
+                <div className="stock-square">
+                    <small>swuare</small>
+                </div>
+            </div>
+        </div>
+    );
+}
+
+export default Trade;
```
Comment (JScheire) [inline frontend/src/components/Trade/Trade.js:143]: Is the all this trading stuff part of another PR?

Comment (JScheire) [reply frontend/src/components/Trade/Trade.js:143]: If so, can we just merge this and close the other one

Comment (aarushiS) [reply frontend/src/components/Trade/Trade.js:143]: yeah we can merge this

**Review by RiyaTayal20 — APPROVED** [review]

**Review by JScheire — APPROVED** [review]

**Review by avidlaud — APPROVED** [review]

**Review by JScheire — APPROVED** [review]

**Pull request merged**